apps/chat/README.md

```diff
@@ -112,7 +112,9 @@ AI DIAL Chat uses environment variables for configuration. All environment varia
-| `ADMIN_ROLE_NAMES`                        |    No    | Defines the name of the administrator role                                                                                                                                                                                                                                                                                    | Any string. Values must be separated by a comma.                                                                                | `admin`                                                                                                                            |
+| `AUTH_{providerId}_ADMIN_ROLE_NAMES`      |    No    | Defines the administrator names for a specific identity provider. `providerId` should be replaced with valid provider id. List of potential provider IDs you may see here: https://next-auth.js.org/providers (choose a provider and check the options section)                                                               | Any string. Values must be separated by a comma.                                                                                |                                                                                                                                    |
+| `AUTH_{providerId}_DIAL_ROLES_FIELD`      |    No    | Defines the path of the roles field in JWT token for a specific identity provider. `providerId` should be replaced with valid provider id.                                                                                                                                                                                    | Any string. Value can be dot-separated. E.g. path `realm_access.roles` if there is a claim `realm_access: { roles: ['admin'] }` |                                                                                                                                    |
+| `ADMIN_ROLE_NAMES`                        |    No    | Defines the names of administrator roles                                                                                                                                                                                                                                                                                      | Any string. Values must be separated by a comma.                                                                                | `admin`                                                                                                                            |
```

Comment:
Move all these env variables include `DIAL_ROLES_FIELD` to auth providers table below and write `AUTH_{providerId}_ADMIN_ROLE_NAMES` and `AUTH_{providerId}_DIAL_ROLES_FIELD` for each provider separately